src-tauri/tauri.conf.json

```diff
@@ -327,6 +327,11 @@
                         "cmd": "whois",
                         "args": true
                     },
+                    {
```

Comment:
I realised that when you run pkexec, you probably do not need to add this as a scoped command. Can you please comment this out and see if the tool runs without it?